QUARKUS-6263.md

```diff
@@ -0,0 +1,49 @@
+# QUARKUS-6263 - Add OIDC Health Check
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6263
+
+Upstream issues: https://github.com/quarkusio/quarkus/issues/47861
+
+Upstream PR: https://github.com/quarkusio/quarkus/pull/47830
+
+## Scope of the testing
+
+- OIDC Health Check is correctly registered when `quarkus.oidc.health.enabled=true` and `quarkus-smallrye-health` dependency is present
+- OIDC Health Check is not registered if `quarkus.health.extensions.enabled=false`
+- Health check correctly reports individual tenant statuses: OK, Not Ready, Disabled, Unknown, and Error
```

Comment:
You can test UP, DOWN plus what happens on the error judging from https://github.com/quarkusio/quarkus/pull/47830/files